astropy/io/misc/asdf/tags/coordinates/skycoord.py

```diff
@@ -26,4 +23,5 @@ def from_tree(cls, tree, ctx):
 
     @classmethod
     def assert_equal(cls, old, new):
+        from astropy.table.tests.test_operations import skycoord_equal
```

Comment:
Please add a comment to indicate why the import is here - otherwise it might get moved back if we do some cleanup one day.